ipaserver/install/adtrust.py

```diff
@@ -451,6 +454,7 @@ class ADTrustInstallInterface(ServiceAdminInstallInterface):
         description="Add IPA masters to a list of hosts allowed to "
                     "serve information about users from trusted forests"
     )
+    add_agents = replica_install_only(add_agents)
```

Comment:
This is just a cosmetic fix - the knob was redefined in `ServerInstallInterface` only to decorate it with `replica_install_only`, which is a) completely unnecessary b) inconsistent with other component installers.